cmd/notary/keys.go

```diff
@@ -338,7 +365,56 @@ func (k *keyCommander) keysImportRoot(cmd *cobra.Command, args []string) error {
 	}
 	defer importFile.Close()
 
-	err = cs.ImportRootKey(importFile)
+	pemBytes, err := ioutil.ReadAll(importFile)
+	if err != nil {
+		return fmt.Errorf("Error reading input file: %v", err)
+	}
+
+	pemRole := trustmanager.ReadRoleFromPEM(pemBytes)
+
+	// Rewind after reading the first time
+	_, err = importFile.Seek(0, 0)
+	if err != nil {
+		return fmt.Errorf("Error reading input file: %v", err)
+	}
+
+	if pemRole != "" && !data.ValidRole(pemRole) {
```

Comment:
Non-blocking - I think I just misinterpreted the behavior somewhat that @diogomonica specified.  I just realized I had previously interpreted "delegation" to mean "either a valid delegation role, or any non-valid role".  I'm fine with imports only handling valid roles, but I was just wondering what we wanted to do for importing user keys that are not limited to specific roles?  Should we do import/export at all, or just have the user drop them in?